### ISSUE TITLE
UnnecessaryAbstractClass: misses class without body

### ISSUE THREAD

**Issue by schalkms** [arturbosch/detekt#550]
The mentioned rule does not detect the following case where an abstract class has no body and therefore no abstract member.

```kotlin
abstract class OnlyConcreteMembersInAbstractClassWithPrimaryCtor2(val i: Int)
```
